src/PitchClass.hs

```diff
@@ -7,20 +7,20 @@ import Chord
 
 
 
-data PitchClass = PitchClass Int
+newtype PitchClass = PitchClass Int
```

Comment:
This wasn't actually a crucial change, but it results in more efficient code. `newtype` essentially tells the compiler that we're defining a type alias, but it should be treated as distinct from the underlying type; the efficiency comes from the compiler's knowledge that the `newtype` and its underlying type (in this case, `Int`) should be represented exactly the same way - there's no additional cost to construct a `PitchClass` even though we need the constructor.